projects/step-frontend/src/lib/modules/execution/components/alt-report-performance-overview-chart/alt-report-performance-overview-chart.component.ts

```diff
@@ -24,23 +29,21 @@ export class AltReportPerformanceOverviewChartComponent {
   };
 
   protected readonly metricKey = 'response-time';
-  protected readonly _mode = inject(VIEW_MODE);
 
   /** @Input() **/
   isFullRange = input<boolean>(true);
 
   /** @Input() **/
   executionId = input.required<string>();
 
-  /** @Input() **/
-  timeRange = input<TimeRange>();
-
   /** @Output() **/
   timeRangeChange = output<TimeRange>();
 
   /** @Output() **/
   fullRange = output();
 
+  activeTimeRange = this._state.timeRangeChange$.pipe(map((s) => s.absoluteSelection!));
```

Comment:
@lucianbaciu 
- If you want to keep it as observable, add $ suffix to the filed name
- As alternate it can be converted to signal, with `toSingal` function
- Please add `protected readonly` modifier.